src/table/pagination-table/components/TableWrapper.tsx

```diff
@@ -48,6 +52,10 @@ function TableWrapper(props: TableWrapperProps) {
     tableContainerRef.current?.scrollTo(0, 0);
   }, [pageInfo, totalRowCount]);
 
+  useLayoutEffect(() => {
+    setBorderHeight(paginationTableRef.current?.getBoundingClientRect?.()?.height);
+  }, [rootElement, rows.length, headHeight]);
```

Comment:
Why `rootElement` here?

Reply:
Should not totals height be included as well?

Reply:
Also, im guessing eslint warns that `paginationTableRef` is missing in the deps array. If so, you should add it.

Reply:
No, eslint does not warn that. 
Yes, totals height should be included as well. I got a new idea then.
I forgot why rootElement is here. it seems unnecessary.